src/main/kotlin/no/nav/familie/ef/sak/service/VedtaksbrevService.kt

```diff
@@ -26,7 +39,12 @@ class VedtaksbrevService(private val brevClient: BrevClient,
     }
 
     fun lagSaksbehandlerBrev(behandlingId: UUID, brevrequest: JsonNode, brevmal: String): ByteArray {
-        // TODO validere at behandlig har rett status/steg?
+        val behandling = behandlingService.hentBehandling(behandlingId)
+        if (behandling.status.behandlingErLåstForVidereRedigering()) {
```

Comment:
Burde man sjekke att man er i riktig steg i stedet? Eks skal man vel ikke generere brev når man er i steget vilkår? 
Ellers vil vi vel risikere att man oppretter ett brev i steget vilkår, vilkår/vedtak blir endret men brevet blir ikke oppdatert med siste dataene?

Reply:
Jeg tenker umiddelbart at det bør være lov å lage brev i vilkår/vedtak stegene. Vi har en state `kanSendesTilBeslutter` i frontend som gjør at  at brevet ikke kan sendes inn før man har trykket "Generer brev".

Argumentet for at brev skal være "åpent" er at vi i fremtiden vil bruke vedtaks/grunnlagsdata direkte, og da kan det være fint å se hvordan det slår ut i brevet uten at man få ferdigstille vedtaket først.

Reply:
Enig. 
Eller 2 faner 🦸 hehe

Reply:
Sant 😬